install.yaml

```diff
@@ -29,7 +29,7 @@ post_install_actions:
     {{ $project_tld := "ddev.site" }}
     {{ if .DdevGlobalConfig.project_tld }}{{ $project_tld = .DdevGlobalConfig.project_tld }}{{ end }}
     {{ if .DdevProjectConfig.project_tld }}{{ $project_tld = .DdevProjectConfig.project_tld }} {{ end }}
-    {{ $novarnish_hostnames := print "novarnish." .DdevProjectConfig.name "." $project_tld  }}
+    {{ $novarnish_hostnames := print "novarnish.\\$DDEV_HOSTNAME" }}
```

Comment:
An option here is to use sprig's `env` function I think, https://masterminds.github.io/sprig/os.html

Reply:
We don't want to expand it (PR is about making it dynamic).

The current behavior (not in this PR) already expands all hostnames, but @FlorentTorregrosa wants it to work even when you change the project name.

The first stopper was that go parser expanded `$DDEV_HOSTNAME` to it's value. I managed to prevent the expansion with `\\$DDEV_HOSTNAME`, so the resulting yaml looks like `VIRTUAL_HOST=novarnish.$DDEV_HOSTNAME`.

Now the problem is that the tests fail, because `novarnish.` is added only to the first hostname in `novarnish.$DDEV_HOSTNAME`.

But we can't do any bash manipulation on the resulting yaml file.

Sadly, I don't see a way to make it work.